src/components/oversikt-krav/KravListe.tsx

```diff
@@ -16,22 +18,25 @@ const KravListe = (props: KravListeProps) => {
     <>
       <Row>
         <Column>
-          <Sidetittel>Oversikt over tidligere innsendte krav</Sidetittel>
+          <Innholdstittel>Oversikt over tidligere innsendte krav</Innholdstittel>
+          <br />
+          <br />
           <Normaltekst>
-            Har du tidligere søkt om refusjon av sykepenger for fravær som skyldes korona eller tapt arbeidsinntekt på
-            grunn av innreiseforbudet, kan du finne tilbake til dem nedenfor.
+            Har du tidligere søkt om refusjon for{' '}
+            <InternLenke to={lenker.Bulkinnsending}>tapt arbeidsinntekt på grunn av innreiseforbudet</InternLenke>, kan
+            du finne tilbake til dem nedenfor.
           </Normaltekst>
+          <br />
           <hr></hr>
-          <ul>
-            {props.innsendinger &&
-              props.innsendinger.map((item) => (
-                <li key={item}>
-                  <Lenke href='#' onClick={() => props.onKravClick(item)}>
-                    {formaterIsoTimestampAsNoTime(item)}
-                  </Lenke>
-                </li>
-              ))}
-          </ul>
+          {props.innsendinger &&
+            props.innsendinger.map((item) => (
+              <div key={item}>
```

Comment:
Hvorfor vil du heller ha mange div'er i stedet for en liste? Det er jo en liste det er ...

Reply:
Grunnet styling, listen får padding. Men kan lave tilbake til liste og override i stedet om det er bedre.

Reply:
Det er semantisk riktig å bruke LI når det faktisk er en liste.